core/test/forest/LocalLinearForestTest.cpp

```diff
@@ -21,9 +21,48 @@
 #include "forest/ForestTrainer.h"
 #include "forest/ForestTrainers.h"
 #include "utilities/ForestTestUtilities.h"
+#include <chrono>
+#include <prediction/LocalLinearPredictionStrategy.h>
+#include <prediction/collector/DefaultPredictionCollector.h>
 
 #include "catch.hpp"
 
+TEST_CASE("LLF_friedman_2k") {
```

Comment:
We try to write test case descriptions in plain English -- I think this one would be 'LLF gives reasonable predictions on data generated from the Friedman function'? Or is this a characterization test (i.e. it just exists to check that predictions have not changed too much as we make performance improvements)?